BurnOutSharp/ProtectionType/AACS.cs

```diff
@@ -12,23 +12,49 @@ public string CheckPath(string path, bool isDirectory, IEnumerable<string> files
         {
             if (isDirectory)
```

Comment:
What happens if you find these files and you're not in a directory?

Reply:
Nothing, and nothing should happen because you will always find a valid AACS protection in the AACS folder.